test/e2e/common/node/pod_level_resources_resize.go

```diff
@@ -0,0 +1,732 @@
+/*
+Copyright 2024 The Kubernetes Authors.
```

Comment:
2025

Reply:
Better yet, just remove the year. It's not needed any more.